bundle/manifests/oadp-operator.clusterserviceversion.yaml

```
@@ -960,4 +960,4 @@ spec:
     name: data-mover-dummy-pod
   - image: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
     name: mustgather
-  version: 99.0.0
+  version: 1.2.0
```

@shubham-pampattiwar @kaovilai , I'd argue this should be v1.2.2 since that's what we're currently building from this branch downstream -- and reflects the same workflow/usage of our other products (such as MTC, for example)
```suggestion
  version: 1.2.2
```